src/dct/enum.lua

```diff
@@ -37,8 +37,8 @@ enum.assetType = {
 	["SHORAD"]      = 19,
 	["AIRBASE"]     = 20,
 	["PLAYERGROUP"] = 21,
-	["SPECIALFORCES"] = 22,
```

Comment:
No asset types cannot be renamed. This would break theaters that use the original names.